modules/dnn/src/onnx/onnx_graph_simplifier.hpp

```diff
@@ -27,10 +27,8 @@ void simplifySubgraphs(opencv_onnx::GraphProto& net);
 template<typename T1, typename T2>
 void convertInt64ToInt32(const T1& src, T2& dst, int size)
 {
-    for (int i = 0; i < size; i++) {
-        if (src[i] < std::numeric_limits<int32_t>::min() || src[i] > std::numeric_limits<int32_t>::max()) {
-            CV_Error(Error::StsOutOfRange, "Input is out of OpenCV 32S range");
-        }
+    for (int i = 0; i < size; i++)
+    {
```

Comment:
Is it necessary for this PR?

Reply:
yeah, tests were failing without this change.